cmd/kube-proxy/app/server.go

```diff
@@ -376,10 +378,14 @@ func (o *Options) Run() error {
 		return cleanupAndExit()
 	}
 
-	proxyServer, err := newProxyServer(o.config, o.master)
+	proxyServer, err := newProxyServer(o.config, o.master, o.InitAndExit)
 	if err != nil {
 		return err
 	}
+	if proxyServer.Proxier == nil {
```

Comment:
Why test this effect rather than testing `o.InitAndExit` directly?

Reply:
Good point, I'll update